src/components/ProductDetailsInfo/index.tsx

```diff
@@ -4,8 +4,8 @@ import ColorSelection from 'components/ColorSelection';
 import AddToBagButton from 'components/AddToBagButton';
 import Accordion from 'components/Accordion';
 import { Size, Color } from 'types';
-import black from '../../assets/images/chooseColor/black.png';
-import white from '../../assets/images/chooseColor/white.png';
+import BlackColor from 'assets/images/chooseColor/black.png';
+import WhiteColor from 'assets/images/chooseColor/white.png';
```

Comment:
It's ok. 
But it seems the variable name doesn't need to include the word Color. Maybe better to name **BlackItem** and **WhiteItem** or **BlackItemColor** and **WhiteItemColor**.